plugins/pastefromword/filter/default.js

```diff
@@ -1156,16 +1156,24 @@
 				List.removeSymbolText( element );
 			}
 
-			if ( element.attributes.style ) {
-				// Hacky way to get rid of margin left.
-				// @todo: we should gather all css cleanup here, and consider bidi. Eventually we might put a config variable to
-				// to enable it.
-				var styles = tools.parseCssText( element.attributes.style );
+			var styles = element.attributes && tools.parseCssText( element.attributes.style );
 
-				if ( styles[ 'margin-left' ] ) {
+			// Default list has 40px padding. To correct indentation we need to reduce margin-left by 40px for each list level.
+			// Additionally margin has to be reduced by sum of margins of each parent, however it can't be done until list are structured in a tree (#2870).
+			// Note margin left is absent in IE pasted content.
+			if ( styles[ 'margin-left' ] ) {
+				var margin = styles[ 'margin-left' ],
+					level = element.attributes[ 'cke-list-level' ];
+
+				margin = Math.max( CKEDITOR.tools.convertToPx( margin ) - 40 * level, 0 );
```

Comment:
I'm not sure about doing so. Currently we preserve Word margins 1:1, if we go back to 40px we will change how pasted document looks like.

For example, when you create list item which is not first, one press on `backspace` will turn current line into paragraph with same indentation level as last list item.

Word document
![Screenshot 2019-05-13 at 11 40 33](https://user-images.githubusercontent.com/23366849/57612051-643ebc80-7574-11e9-9c92-e46cb4676529.png)

Editor output when subtracting `40px`
![Screenshot 2019-05-13 at 11 43 32](https://user-images.githubusercontent.com/23366849/57612112-7de00400-7574-11e9-8ed8-4fc2ff55d2af.png)

Editor output when subtracting `48px`
![Screenshot 2019-05-13 at 11 43 37](https://user-images.githubusercontent.com/23366849/57612135-86383f00-7574-11e9-9c32-fb99e2ef7d42.png)

If we wan't to pasted Word document to keep it's look and structure we need to remove default html styles (`40px`), otherwise they will interfere with pasted content.

Note: examples are pasted with `allowedContent: true`, however `extraAllowedContent: 'li{margin-left};p{margin-left}'` would be sufficient, it would just look ugly with bullets not removed from higher order lists.

Reply:
☝️ Answering my own questions 😅

Reply:
Just to clarify. I understand, that you don't expect any further changes here.

Reply:
Yes 👌